frontends/mit-open/src/page-components/Header/Header.tsx

```diff
@@ -29,26 +45,124 @@ const StyledDivider = styled(Divider)({
   margin: "0.5em 1em",
 })
 
-const BrandLink = styled(Link)`
-  font-weight: bold;
-  color: ${({ theme }) => theme.palette.secondary.main};
-`
-
 const Spacer = styled.div`
   flex: 1;
 `
 
+const navData: NavData = {
+  sections: [
+    {
+      title: "LEARN",
+      items: [
+        {
+          title: "Courses",
+          description: "Learn with MIT instructors",
+          href: querifiedSearchUrl({ resource_type: "course" }),
+        },
+        {
+          title: "Programs",
+          description:
+            "Learn in-depth from a series of courses and earn a certificate",
+          href: querifiedSearchUrl({ resource_type: "program" }),
+        },
+        {
+          title: "Course Materials",
+          description:
+            "Free teaching and learning materials including videos, podcasts, lecture notes, etc.",
+        },
+      ],
+    },
+    {
+      title: "BROWSE",
+      items: [
+        {
+          title: "By Subject",
+        },
+        {
+          title: "By Departments",
+          href: DEPARTMENTS,
+        },
+        {
+          title: "By Provider",
+        },
+      ],
+    },
+    {
+      title: "DISCOVER LEARNING RESOURCES",
+      items: [
+        {
+          title: "New",
+          href: querifiedSearchUrl({
+            resource_type: "course",
+            sortby: "new",
+          }),
+        },
+        {
+          title: "Upcoming",
+          href: querifiedSearchUrl({
+            resource_type: "course",
+            sortby: "upcoming",
+          }),
+        },
+        {
+          title: "Popular",
+          href: querifiedSearchUrl({
+            resource_type: "course",
+            sortby: "popular",
+          }),
+        },
+        {
+          title: "Free",
+          href: querifiedSearchUrl({ free: "true" }),
+        },
+        {
+          title: "With Certificate",
+          href: querifiedSearchUrl({ certification: "true" }),
+        },
+      ],
+    },
+  ],
+}
+
 const Header: FunctionComponent = () => {
+  const [drawerOpen, toggleDrawer] = useToggle(false)
+  const [searchParams] = useSearchParams()
+  const resourceDrawerOpen = searchParams.has(RESOURCE_DRAWER_QUERY_PARAM)
+
+  const toggler = (event: React.MouseEvent) => {
+    if (!resourceDrawerOpen) {
+      event.stopPropagation()
+    }
+    toggleDrawer(!drawerOpen)
+  }
```

Comment:
Yea it would be nice if you could add exceptions to `ClickAwayHandler` but in the absence of functionality like that, this seemed like the best solution.